awx/ui_next/src/components/MultiButtonToggle/MultiButtonToggle.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+import styled from 'styled-components';
+import { Button } from '@patternfly/react-core';
+import ButtonGroup from './ButtonGroup';
+
+const SmallButton = styled(Button)`
+  padding: 3px 8px;
+  font-size: var(--pf-global--FontSize--xs);
+`;
+
+function MultiButtonToggle({ buttons, currentValue, onChange }) {
```

Comment:
I'd probably go with `value` as the prop name over `currentValue`, just because that's a common enough convention in other components